src/modules/alwaysontop/AlwaysOnTop/AlwaysOnTop.cpp

```diff
@@ -484,6 +485,20 @@ void AlwaysOnTop::HandleWinHookEvent(WinHookEvent* data) noexcept
         RefreshBorders();
     }
     break;
+    case EVENT_OBJECT_FOCUS:
+    {
+        for (const auto& [window, border] : m_topmostWindows)
+        {
+            // check if topmost was reset
+            // fixes https://github.com/microsoft/PowerToys/issues/19168
+            if (!IsTopmost(window))
+            {
+                Logger::trace(L"Reset topmost");
```

Comment:
Perhaps a more descriptive message? Like "A window no longer has Topmost set and it should. Setting topmost again."